cmd/fyne/package.go

```diff
@@ -14,7 +14,7 @@ import (
 	"strings"
 
 	"fyne.io/fyne"
-	"github.com/Kodeworks/golang-image-ico"
+	ico "github.com/Kodeworks/golang-image-ico"
```

Comment:
I notice my IDE added  the declarative here, it uses the `ico` namespace in implementation anyways so it should not be an issue